core/os/os_linux.odin

```diff
@@ -265,16 +291,27 @@ foreign libc {
 	@(link_name="write")            _unix_write         :: proc(fd: Handle, buf: rawptr, size: c.size_t) -> c.ssize_t ---;
 	@(link_name="lseek64")          _unix_seek          :: proc(fd: Handle, offset: i64, whence: c.int) -> i64 ---;
 	@(link_name="gettid")           _unix_gettid        :: proc() -> u64 ---;
+	@(link_name="getuid")           getuid              :: proc() -> uid_t ---;
```

Comment:
`_unix_getuid` ?

Reply:
There didn't need to be a wrapper around getuid, which is why I didn't have both _unix_getuid and getuid like I did with the others.